optimize/backend/src/test/java/io/camunda/optimize/service/util/DurationFormatterUtilTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class DurationFormatterUtilTest {
+
+  @Test
+  void shouldReturnReadableStringForDurationInMilliseconds() {
+    // given
+    final long durationInMs = 4000000000L; // 46 days, 6 hours, 40 minutes
+
+    // when
+    final String result =
+        DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);
+
+    // then
+    assertThat(result).isEqualTo("1mo 2wks 2d 7h 6min 40s");
+  }
+
+  @Test
+  void shouldReturnDashForZeroOrNegativeDuration() {
+    // given
+    long durationInMs = 0;
+
+    // when
+    String result = DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);
+
+    // then
+    assertThat(result).isEqualTo("-");
+
+    // given
+    durationInMs = -1000;
+
+    // when
+    result = DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);
+
+    // then
+    assertThat(result).isEqualTo("-");
+  }
```

Comment:
```suggestion
  @ParameterizedTest
  @ValueSource(ints = {0, -1000})
  void shouldReturnDashForZeroOrNegativeDuration(final int durationInMs) {
    // when
    String result = DurationFormatterUtil.formatMilliSecondsToReadableDurationString(durationInMs);

    // then
    assertThat(result).isEqualTo("-");
  }
```

NABD but I like this very succinct way of testing with parameters

Reply:
fixed